test/tests.js

```diff
@@ -0,0 +1,73 @@
+//import './example.test.js';
+
```

Comment:
I'm a little confused by these tests, you seem to have written a different function for every possibility rather than testing one compareNumbers functions. You also don't seem to use these functions anywhere in your app.js.